ledger/light/pulsemanager/pulsemanager.go

```diff
@@ -421,6 +424,11 @@ func (m *PulseManager) Set(ctx context.Context, newPulse insolar.Pulse, persist
 		go m.LightReplicator.NotifyAboutPulse(ctx, newPulse.PulseNumber)
 	}
 
+	err = m.WriteManager.Open(ctx, newPulse.PulseNumber)
+	if err != nil {
```

Comment:
We should log the error and move on. We can't stop all the processing because of this.

Reply:
done